grpclb/src/test/java/io/grpc/grpclb/GrpclbLoadBalancerTest.java

```diff
@@ -601,15 +601,20 @@ public void grpclbWorking() {
     assertSame(error1, picker6.result.getStatus());
 
     // Update backends, with a drop entry
-    List<InetSocketAddress> backends2 = Arrays.asList(
-        new InetSocketAddress("127.0.0.1", 2030), null);
+    List<InetSocketAddress> backends2 =
```

Comment:
I agree the state of this test method is not ideal. Although it's testing multiple behaviors, every test depends the state left by the the previous tests as prerequisites, therefore it's non-trivial to split them apart, and I don't have the time to do it.